moto/rds/models.py

```diff
@@ -45,6 +49,61 @@
 )
 
 
+def find_cluster(cluster_arn):
+    arn_parts = cluster_arn.split(":")
+    region, account = arn_parts[3], arn_parts[4]
+    return rds_backends[account][region].describe_db_clusters(cluster_arn)[0]
+
+
+class GlobalCluster(BaseModel):
+    def __init__(
+        self,
+        account_id: str,
+        global_cluster_identifier: str,
+        engine: str,
+        engine_version,
```

Comment:
nit: type hints

Reply:
Yeah.. I know. I went back and forth on that. I decided against it in the end, because the PR was big enough already.
I'm planning to go back in later on to add type hints everywhere (and turn MyPy on for the RDS-module - not all sections are currently covered)

https://github.com/getmoto/moto/blob/1111e10b875af3bc69552be0f16b116edb8eb629/setup.cfg#L238